regression/global_3dvar.sh

```diff
@@ -0,0 +1,306 @@
+#   satangl  = angle dependent bias correction file (fixed in time)
+#   pcpinfo  = text file with information about assimilation of prepcipitation rates
+#   ozinfo   = text file with information about assimilation of ozone data
+#   errtable = text file with obs error for conventional data (optional)
+#   convinfo = text file with information about assimilation of conventional data
+#   bufrtable= text file ONLY needed for single obs test (oneobstest=.true.)
+#   bftab_sst= bufr table for sst ONLY needed for sst retrieval (retrieval=.true.)
+#   aeroinfo = text file with information about assimilation of aerosol data
+
+anavinfo=$fixgsi/global_anavinfo.l${LEVS}.txt
+berror=$fixgsi/Big_Endian/global_berror.l${LEVS}y${NLAT}.f77
+locinfo=$fixgsi/global_hybens_info.l${LEVS}.txt
+satinfo=$fixgsi/global_satinfo.txt
+scaninfo=$fixgsi/global_scaninfo.txt
+satangl=$fixgsi/global_satangbias.txt
+pcpinfo=$fixgsi/global_pcpinfo.txt
+ozinfo=$fixgsi/global_ozinfo.txt
+convinfo=$fixgsi/global_convinfo.txt
+vqcdat=$fixgsi/vqctp001.dat
+insituinfo=$fixgsi/global_insituinfo.txt
+errtable=$fixgsi/prepobs_errtable.global
+aeroinfo=$fixgsi/global_aeroinfo.txt
+atmsbeaminfo=$fixgsi/atms_beamwidth.txt
+cloudyinfo=$fixgsi/cloudy_radiance_info.txt
+
+emiscoef_IRwater=$fixcrtm/Nalli.IRwater.EmisCoeff.bin
+emiscoef_IRice=$fixcrtm/NPOESS.IRice.EmisCoeff.bin
+emiscoef_IRland=$fixcrtm/NPOESS.IRland.EmisCoeff.bin
+emiscoef_IRsnow=$fixcrtm/NPOESS.IRsnow.EmisCoeff.bin
+emiscoef_VISice=$fixcrtm/NPOESS.VISice.EmisCoeff.bin
+emiscoef_VISland=$fixcrtm/NPOESS.VISland.EmisCoeff.bin
+emiscoef_VISsnow=$fixcrtm/NPOESS.VISsnow.EmisCoeff.bin
+emiscoef_VISwater=$fixcrtm/NPOESS.VISwater.EmisCoeff.bin
+emiscoef_MWwater=$fixcrtm/FASTEM6.MWwater.EmisCoeff.bin
+aercoef=$fixcrtm/AerosolCoeff.bin
+cldcoef=$fixcrtm/CloudCoeff.bin
+#cldcoef=$fixcrtm/CloudCoeff.GFDLFV3.-109z-1.bin   # use with crtm/2.4.0
+
+# Only need this file for single obs test
+bufrtable=$fixgsi/prepobs_prep.bufrtable
+
+# Only need this file for sst retrieval
+bftab_sst=$fixgsi/bufrtab.012
+
+# Copy executable and fixed files to $tmpdir
+if [[ $exp == *"updat"* ]]; then
+   $ncp $gsiexec_updat  ./gsi.x
+elif [[ $exp == *"contrl"* ]]; then
+   $ncp $gsiexec_contrl ./gsi.x
+fi
+
+$ncp $anavinfo ./anavinfo
+$ncp $berror   ./berror_stats
+$ncp $locinfo  ./hybens_info
+$ncp $satinfo  ./satinfo
+$ncp $scaninfo ./scaninfo
+$ncp $pcpinfo  ./pcpinfo
+$ncp $ozinfo   ./ozinfo
+$ncp $convinfo ./convinfo
+$ncp $vqcdat   ./vqctp001.dat
+$ncp $insituinfo ./insituinfo
+$ncp $errtable ./errtable
+$ncp $aeroinfo ./aeroinfo
+$ncp $atmsbeaminfo ./atms_beamwidth.txt
+$ncp $cloudyinfo   ./cloudy_radiance_info.txt
+
+$ncp $bufrtable ./prepobs_prep.bufrtable
+$ncp $bftab_sst ./bftab_sstphr
+
+#If using correlated error, get the covariance files
+if grep -q "Rcov" $anavinfo ;
+then
+  if ls ${fixgsi}/Rcov* 1> /dev/null 2>&1;
+  then
+    $ncp ${fixgsi}/Rcov* $DATA
```

Comment:
Where is `$DATA` defined? is this supposed to be `$tmpdir`?

Reply:
I see that it is defined in the submission script, but is this copying to the right spot then?

Reply:
You caught a mistake.   `$DATA` should be `$tmpdir`.   Corrected at 1ba3fb54.